resources/views/auth/register.blade.php

```diff
@@ -75,6 +75,18 @@
                             </div>
                         </div>
 
+                        <div class="form-group row">
+                            <label for="role_id" class="col-md-4 col-form-label text-md-right"> Role </label>
+                            <div class="col-md-6">
+                                <select name="role_id" id="role_id">
+                                    @foreach($roles as $role)
+                                        <option value="{{$role->id}}" {{old('role_id', $role->id)}}? selected> {{$role->name}} </option>
```

Comment:
This doesn't work: `{{old('role_id', $role->id)}}?`.

See: https://github.com/b019-g13/Sirkus-Media-nettsted-back-end/blob/master/resources/views/pages/form-fields.blade.php#L7-L22 for how this can be done